WebApp/Viewer.aspx.cs

```diff
@@ -1684,4 +1697,56 @@ private void ShowLevelSelector(Configuration.ApplicationRow application)
       }
     }
   }
-}
\ No newline at end of file
+
+
+  //New code for BaseLayer
```

Comment:
If not in use remove this commented code

Reply:
done